roles/devscripts/tasks/133_host_network.yml

```diff
@@ -88,7 +93,7 @@
     - name: Create the virtual networks.
       vars:
         cifmw_libvirt_manager_net_prefix_add: false
-        _layout:
+        _cifmw_libvirt_manager_layout:
```

Comment:
@psathyan apparently I missed that one while working on the renaming - not sure how to get through that path though... IIRC that's for hybrid?

Reply:
This path is taken when user has choosen to set `manage_br_bridge: 'n'` ... generally the path is taken during hybrid scenarios for ensuring a physical port is attached to the bridge.

Thanks @cjeanner for fixing it. With the latest patch set, I did hit the issue with public networks.